packages/graphql-yoga/__tests__/400.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { createSchema } from '../src/schema'
+import { createYoga } from '../src/server'
+import { DateTimeResolver, DateTimeTypeDefinition } from 'graphql-scalars'
+import { expect } from '@jest/globals'
```

Comment:
I think it should work without needing to import this 
```suggestion
```

Reply:
Oh, I don't think I've imported that. It got auto imported for me

Reply:
But yes, there's no need